doc/source/whatsnew/v0.21.1.txt

```diff
@@ -77,6 +77,7 @@ I/O
 
 - Bug in class:`~pandas.io.stata.StataReader` not converting date/time columns with display formatting addressed (:issue:`17990`). Previously columns with display formatting were normally left as ordinal numbers and not converted to datetime objects.
 - Bug in :func:`read_csv` when reading a compressed UTF-16 encoded file (:issue:`18071`)
+- Bug in ``DataFrame.to_csv(...)`` when the table had ``MultiIndex`` columns, and a list of strings was passed in for ``header`` (:issue:`5539`)
```

Comment:
Not strictly necessary, but might be nice to do <code>:meth:\`DataFrame.to_csv\`</code> to provide a link.

Reply:
@gfyoung can you update this

Reply:
Looks like there was actually only one place (mine) which needed this fix.